integration-tests/src/test/resources/bash-scripts/syncTestRepo.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/up
+
+SOURCE_REPO=${BASE_IMAGES_REPO}
+SOURCE_USER=${BASE_IMAGES_REPO_USERNAME}
+SOURCE_PASSWORD=${BASE_IMAGES_REPO_PASSWORD}
```

Comment:
After your REPO changes PR is done, we will still reference OCR(or)OCIR for base images? If yes we should use OCR as the BASE_IMAGES_REPO here.